### PR TITLE
Add Xevunleashed

### PR DESCRIPTION
**Pull request by JackDawson94**
Add scraper for https://xevunleashed.com/

Works as intented in all my tests.

Only thing I couldn't get better was the list of performers.
Unfortunately the performers are never listed in the page itself, only in the tags...

So either:
1. performers list = tags list 
2. no performers

I chose option 1 (similar to existing behaviour in Clips4sale scraper)

### CODE REVIEW

File: scrapers/Xevunleashed.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      Studio:
+        Name:
+          fixed: XevUnleashed
+      Image: //meta[@name="twitter:image"]/@content
+      Tags:
+        Name: //span[@class="update_tags"]//a
+      Performers:
+        Name:
+          selector: //meta[@name="keywords"]/@content
+          split: ','
```
Comment (bnkai) [inline scrapers/Xevunleashed.yml:31]: Where there scenes where the performers are mentioned in the keywords? Even `Xev Bellringer` isnt mentioned in any of the scenes i tried

File: scrapers/Xevunleashed.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      Title: //span[@class="update_title"]/text()
+      Date:
+        selector: //span[@class="availdate"]/text()
+        concat: " "
+        postProcess:
+          - replace:
+              - regex: (\d{2}\/\d{2}\/\d{4}).*
+                with: $1
+          - parseDate: 01/02/2006
+      Details: //meta[@name="description"]/@content
```
Comment (bnkai) [inline scrapers/Xevunleashed.yml:21]: For long descriptions this only get's us partial meta.
```suggestion
      Details: //span[@class="latest_update_description"]
```
seems to work ok

File: scrapers/Xevunleashed.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    scene:
+      Title: //span[@class="update_title"]/text()
+      Date:
+        selector: //span[@class="availdate"]/text()
+        concat: " "
+        postProcess:
+          - replace:
+              - regex: (\d{2}\/\d{2}\/\d{4}).*
+                with: $1
+          - parseDate: 01/02/2006
```
Comment (bnkai) [inline scrapers/Xevunleashed.yml:20]:
```yaml
        selector: //span[@class="availdate"]/text()[1]
        postProcess:
          - parseDate: 01/02/2006
                 selector: //span[@class="availdate"]/text()[1]
        postProcess:
          - parseDate: 01/02/2006
  ```
Should be enough
The first text is always the date part

File: scrapers/Xevunleashed.yml (hunk 1 above)
Comment (JackDawson94) [reply scrapers/Xevunleashed.yml:31]: It gets them when there are 2 performers (so like... less than 10 scenes overall I think)

Not sure if it's worth it...

File: scrapers/Xevunleashed.yml (hunk 3 above)
Comment (JackDawson94) [reply scrapers/Xevunleashed.yml:20]: Nice, didn't know that worked, will update asap

File: scrapers/Xevunleashed.yml (hunk 1 above)
Comment (bnkai) [reply scrapers/Xevunleashed.yml:27]: Maybe add Xev Bellringer as a fixed name performer?

Comment (JackDawson94) [reply scrapers/Xevunleashed.yml:27]: Can you share a couple of links to scenes where this didn't work ? I tried to find some but couldn't find any where her name isn't in the keywords

Comment (bnkai) [reply scrapers/Xevunleashed.yml:27]: Hm... i think it was the few scenes i tested.
I tested mostly scenes from here to see if any extra performers were in the keywords... <https://xevunleashed.com/categories/mnage--trois.html>

Comment (bnkai) [reply scrapers/Xevunleashed.yml:27]: @JackDawson94 i thinks its ok to merge as is ? Is it ok to merge now?

Comment (JackDawson94) [reply scrapers/Xevunleashed.yml:27]: Ok to merge yeah

**Pull request merged**